jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/python/PythonDebugUtils.kt

```diff
@@ -33,6 +37,9 @@ object PythonDebugUtils {
                 )
             )
 
+            val sdk = ProjectRootManager.getInstance(environment.project).projectSdk?.takeIf { it.sdkType is PythonSdkType }
```

Comment:
can the module have a different sdk than the project?

Reply:
Ya in IntelliJ

Guess we need to find a way to get the Module down here if we can..

Reply:
We dont really have a way to get the module that far down thats event remote clean :/

Reply:
Can you add a TODO in here for that?